lib/config/ChainConfigManager.ts

```diff
@@ -0,0 +1,293 @@
+import { ChainId } from '@uniswap/sdk-core';
+import { RoutingType } from '../constants';
+
+type ChainConfig = {
+  routingTypes: (
+    | {
+        routingType: RoutingType.CLASSIC | RoutingType.DUTCH_V2 | RoutingType.RELAY;
+        skipRFQ?: boolean;
+        priceImprovementBps?: number;
+        stdAuctionPeriodSecs?: number;
+        deadlineBufferSecs?: number;
+      }
+    | {
+        routingType: RoutingType.DUTCH_LIMIT;
+        skipRFQ?: boolean;
+        priceImprovementBps?: number;
+        stdAuctionPeriodSecs?: number;
+        lrgAuctionPeriodSecs?: number;
```

Comment:
Hmm this conditional logic seems like something that we'd want to configure on a more fine grained level than just the routing type - like for example, the threshold for a large vs. small trade may be different on mainnet than arbitrum. So I think this would work better within the request entity?

Reply:
It's chain and routing type so it can handle the scenario you described. See the `_chainConfigs` below.